spec/till_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'till'
+require 'timecop'
+
+describe Till do
+
+  let(:till) { described_class.new( { shopName: "The Coffee Connection",
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.